DashDocs/src/chapters/dash_vtk/RepresentationComponents/index.jl

```diff
@@ -0,0 +1,351 @@
+
+            - *size*: width, height in pixel for the controller UI
+            - *rescaleColorMap*: true/false to use the opacity piecewise function to dynamically rescale the color map or keep the full data range as color range.
+
+
+        ## vtk_slice_representation
+
+        The `vtk_slicerepresentation` lets you see a slice within a 3D image. That slice can be along i,j,k or x,y,z if your volume contains an orientation matrix.
+
+        The following set of properties lets you pick which slice you want to see. Only one of those properties can be used at a time.
+
+            - `iSlice`, `jSlice`, `kSlice`: Index based slicing
+            - `xSlice`, `ySlice`, `zSlice`: World coordinate slicing
+
+        Then we have the standard representation set or properties with their defaults:
+
+            - `actor`:
+                       - `origin` = (0,0,0)
+                       - `position` = (0,0,0)
+                       - `scale` = (1,1,1)
+                       - `visibility` = 1
+                       - `pickable` = 1
+                       - `dragable` = 1
+                       - `orientation` = (0,0,0)
+
+            - `property`:
+                          - `independentComponents` = false
+                          - `interpolationType` = [InterpolationType.LINEAR](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/ImageProperty/Constants.js#L1-L4)
+                          - `colorWindow` = 255
+                          - `colorLevel` = 127.5
+                          - `ambient` = 1.0
+                          - `diffuse` = 0.0
+                          - `opacity` = 1.0
+             
+            - `mapper`:
+                        - `customDisplayExtent`: 0, 0, 0, 0
+                        - useCustomExtents: false
+                        - slice: 0
+                        - slicingMode: [SlicingMode.NONE](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/ImageMapper/Constants.js#L1-L9)
+                        - closestIJKAxis: { ijkMode: [SlicingMode.NONE](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/ImageMapper/Constants.js#L1-L9), flip: false }
+                        - renderToRectangle: false
+                        - sliceAtFocalPoint: false  
+
+        
+        ## vtk_pointcloudrepresentation
+        
+        The `vtk_pointcloudrepresentation` is just a helper using the following structure to streamline rendering a point cloud dataset. The code snippet below is not complete but it should provide you with some understanding of the kind of simplification that is happening under the hood.
+
+        ```
+        function vtk_pointcloudrepresentation(; kwargs...) 
+                 return vtk_geometryrepresentation(
+                     id=kwargs.get("id"),
+                     colorMapPreset=kwargs.get("colorMapPreset"),
```

Comment:
Using kwargs.get like that is not idiomatic in Julia. You probably want to do `kwargs[:colorMapPreset]`

Reply:
Please address this in a separate PR.